frappeclient/frappeclient.py

```diff
@@ -1,11 +1,14 @@
 import requests
 import json
 
-
+try:
+	from urllib.parse import quote
+except:
+	from urllib import pathname2url as quote
```

Comment:
there is quote in python2 urllib:

```suggestion
	from urllib import quote
```